src/hotspot/share/c1/c1_LIR.hpp

```diff
@@ -1701,6 +1692,63 @@ class LIR_Op2: public LIR_Op {
   virtual void print_instr(outputStream* out) const PRODUCT_RETURN;
 };
 
+#ifdef RISCV
```

Comment:
Would you be able not to move this code? Leave it in place, so that diff is cleaner?

Reply:
Hi, I am afraid that this has to come after class definition of `LIR_Op2` as class `LIR_OpBranch` extends public `LIR_Op2` for RISC-V. Otherwise, I got compile errors like:
```
In file included from /home/fyang/jdk17u-dev-fork/src/hotspot/share/c1/c1_Instruction.hpp:29,
                 from /home/fyang/jdk17u-dev-fork/src/hotspot/share/c1/c1_LIRGenerator.hpp:29,
                 from /home/fyang/jdk17u-dev-fork/src/hotspot/share/gc/shared/c1/barrierSetC1.cpp:27:
/home/fyang/jdk17u-dev-fork/src/hotspot/share/c1/c1_LIR.hpp:1427:28: error: invalid use of incomplete type 'class LIR_Op2'
 1427 | class LIR_OpBranch: public LIR_Op2 {
      |                            ^~~~~~~
/home/fyang/jdk17u-dev-fork/src/hotspot/share/c1/c1_LIR.hpp:868:10: note: forward declaration of 'class LIR_Op2'
  868 | class    LIR_Op2;
      |          ^~~~~~~
```

Reply:
Aw. Pity. Nevermind then. (Shakes fists at C/C++)